components/scream/cmake/CprncTest.cmake

```diff
@@ -0,0 +1,30 @@
+# Dump vars that are expected to be equal and those expected to be different
```

Comment:
Since #1289 already introduces this file, I think it's best if you rebase this PR on top of that. That way, you'd only see mods _specific_ to this PR.